test/e2e/tests/test_external_models.py

```
@@ -401,21 +487,27 @@ def test_wrong_model_in_body_rejected(self, external_models_setup):
             "messages": [{"role": "user", "content": "hello"}],
         })
         assert r.status_code != 200, (
-            f"Expected rejection for wrong model in body, got 200. "
+            f"Expected non-200 for wrong model in body, got 200. "
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)